module/rental_module.rb

```diff
@@ -0,0 +1,46 @@
+module RentalDisplay
+  def create_rental
+    create_rental_book
+    book_index = gets.chomp.to_i
+    puts
+    create_rental_person
+    person_index = gets.chomp.to_i
+    puts
+    print 'Date: '
+    date = gets.chomp
+    @rentals << Rental.new(date, @books[book_index], @people[person_index])
+    puts 'Rental created successfully'
+  end
+
+  def create_rental_book
+    if @books.empty?
+      puts 'There are no books in the library to rent'
+      return
+    end
+    puts 'Select a book from the following list by number'
+    @books.each_with_index do |book, index|
+      puts "#{index}) Title: '#{book.title}', Author: #{book.author}"
+    end
+  end
+
+  def create_rental_person
+    puts 'Select a person from the following list by number (not id)'
+    if @people.empty?
+      puts 'There are no people in the library'
+      return
+    end
+
+    @people.each_with_index do |person, index|
+      puts "#{index}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
```

Comment:
- [ ] Even I had created the person there wasn't person saved, please fix the issue to create a rental successfully and list them 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/231827499-ff9dba01-ea96-46f4-840d-41633fea3cde.png)

Reply:
Rental also fix